lang/extract_json_strings.py

```diff
@@ -21,7 +21,62 @@ def __str__(self):
     "LOADING_ORDER.md"
 }
 
-# don't parse this files. Full related path.
+# these are official mods need to be localized
+# list could be generated by bash command: (in data/mods directory)
+# for i in $(ls -l1); do echo -n '    "'; echo -n $i; echo '",'; done
+# and remove things like replacements.json manually
+official_mods = {
+    "Animatronics",
```

Comment:
This will become outdated quickly. It would best if you could actually parse `modinfo.json` and look for either `obsolete` or lack of `maintainers`. We don' want to burden the translation team with work of marginal or nil value